### ISSUE TITLE
[Lab3] Semeval dataset Loss function

### ISSUE THREAD

**Issue by chrisdim** [slp-ntua/slp-labs#62]
Καλησπέρα,
Θα ήθελα να ρωτήσω αν γίνεται για δική μας διευκόλυνση να χρησιμοποιήσουμε και στο Semeval2017 dataset πάλι το BCEwithlogits loss παρόλο που έχουμε 3 κλάσεις, καθώς αντιμετωπίζω προβλήματα στην είσοδο του CrossEntropyLoss. 
Ευχαριστώ

**Comment by pxaris** [comment]
Καλύτερα όχι, προτίμησε το `CrossEntropyLoss` που εφαρμόζει softmax στα logits των 3 κλάσεων. Ο ενδεικτικός κώδικας που δόθηκε κατά την [παρουσίαση](https://drive.google.com/file/d/1brMcP5PvdSx19vqgtG-vq-8ItD-2Aeqt/view) του εργαστηρίου (βλ. 3η διαφάνεια) θα πρέπει να είναι αποδοτικός για το loss αυτό.
